api-docs/api-operations/methods/patch-remove-configuration-group-from-ha-instance-version-accountid-ha-haid.rst

```diff
@@ -0,0 +1,150 @@
+
+.. _patch-remove-configuration-group-from-ha-instance-version-accountid-ha-haid.rst:
+
+Remove Configuration Group from HA Instance 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code::
+
+    PATCH /{version}/{accountId}/ha/{haId}
+
+Detach configuration group from the HA Instance.
+
+.. note::
+   If the configuration group has non-dynamic configuration parameters, the HA instance will be put in a ``RESTART_REQUIRED`` state. To disable the parameters on all the nodes (source and replicas of the HA group), restart the HA instance (see :ref:`post-restart-ha-instance-version-accountid-ha-haid-action`.).
+
```

Comment:
:ref:`restart the HA instance<post-restart-ha-instance-version-accountid-ha-haid-action>`.

Remove the "See" reference.